src/client/client.ts

```diff
@@ -31,7 +31,7 @@ import {
 } from '../task';
 import { Omit, RecursiveRequired } from '../types/internal';
 import batchIterator from '../utils/batchIterator';
-import computeNextRun from '../utils/computeNextRun';
+import computeNextRun, { isValidNextRun } from '../utils/computeNextRun';
```

Comment:
nit: the file is named for `computeNextRun`. now that `isValidNextRun` is exported, might make sense to split it into its own file